packages/app/screens/feed/FeedScreen.tsx

```diff
@@ -11,6 +11,30 @@ import { useUserContext } from "@app/utils/UserContext";
 import { logoutUser } from "@app/utils/datalayer";
 import { ENDPOINT } from "@app/utils/constants";
 
+type EventType = {
+  event_id: string,
+  guild_id: string,
+  title: string,
+  description: string,
+  start_date: number,
+  end_date: number,
+  location: string,
+  thumbnail: string
+}
+
+type ItemType = {
+  banner: ImageSourcePropType,
+  title: string,
+  description: string,
+  location: string,
+  start_date: string,
+  end_date: string
+}
```

Comment:
Are these types going to be used only in the feed screen? Maybe these should be put in a type declaration file and then imported to be used in other files as well.

Reply:
Issue should be resolved in the latest commit. Created `@app/types` directory to store all types